docs/sites.yml

```
@@ -10931,3 +10931,20 @@
   built_by: Nagarjun Palavalli
   built_by_url: https://twitter.com/palavalli
   featured: false
+- title: JAMstack Storefront
```

```suggestion
- title: Jamstack Storefront
```

JAMstack is now just Jamstack. As well, it may be good to update such on your site.